spec.md

```diff
@@ -1380,10 +1463,10 @@ For success responses, the following fields are defined:
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| credentials | object | A free-form hash of credentials that can be used by applications or users to access the service. |
-| syslog_drain_url | string | A URL to which logs MUST be streamed. `"requires":["syslog_drain"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the Platform MUST consider the response invalid. |
-| route_service_url | string | A URL to which the Platform MUST proxy requests for the address sent with `bind_resource.route` in the request body. `"requires":["route_forwarding"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the Platform can consider the response invalid. |
-| volume_mounts | array-of-objects | An array of configuration for mounting volumes. `"requires":["volume_mount"]` MUST be declared in the [Catalog](#catalog-management) endpoint or the Platform can consider the response invalid. |
+| credentials | object | A free-form hash of credentials that can be used by applications or users to access the service. MUST be returned if the Service Broker supports generation of credentials and the Service Binding was provisioned asynchronously. |
```

Comment:
I can see this new line being a point of contention. What about brokers that don't want to hold onto credentials or only want to return them once for security issues? I know we've discussed this before but can't remember what we all agreed on?

Reply:
> return them once

@mattmcneeney I don't think it's realistic to "return only once". What about retries from the platform side?

If brokers want to remain stateless, they still have an option of creating binding synchronously. But even in that case create request with unchanged parameters should lead to `200 OK` response with implied unchanged credentials (rather than generating new credentials on retry) according to the spec.